src/main/java/frc/robot/commands/AutoPickupNote.java

```diff
@@ -0,0 +1,124 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+
+import frc.robot.AutoRotateUtil;
+import edu.wpi.first.math.controller.PIDController;
+import frc.robot.subsystems.Swerve;
+import frc.robot.subsystems.SkyLimelight;
+import frc.robot.Constants;
+import frc.robot.subsystems.IntakeSubsystem;
+import edu.wpi.first.math.filter.Debouncer;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+
+public class AutoPickupNote extends Command{
+    public SkyLimelight limelight;
+    public Swerve swerve;
+    public IntakeSubsystem intakeSubsystem;
+    public Debouncer debouncer;
+    public Rotation2d yaw;
+    private AutoRotateUtil autoUtil;
+
+    public PIDController yTranslationPidController, xTranslationPidController, rotationPidController;
```

Comment:
Are we using rotationPidController anywhere? We should be using autoUtil for the rotation PID, so should probably remove this.

Reply:
removed rotationPidController